1116/en.md

```diff
@@ -0,0 +1,51 @@
+# LOJ 1116 - Ekka Dokka #
+---
+
+Input starts with an integer _T (≤ 10000)_, denoting the number of test cases.
+
+Each case contains an integer _W (2 ≤ W < 2<sup>63</sup>)_. And W will not be a power of 2.
+
+For each case, print the case number first. After that print `Impossible` if they can't buy their desired cake. If they can buy such a cake, you have to print `N` and `M`. If there are multiple solutions, then print the result where **M is as small as possible**.
+
```

Comment:
For future reference, declaring a header ("Solution" maybe) after the problem summary and before the solution texts helps some readers.

Reply:
Header has been added.